src/ticketmap/index.tsx

```diff
@@ -42,6 +42,13 @@ interface ElementProperties {
 
 type PropertiesForElement = (element: Element) => ElementProperties
 
+class MapNotFoundError extends Error {
+  name = 'MapNotFoundError'
+  constructor(message = 'This map is not currently available.') {
+    super(message)
```

Comment:
I'm like John Travolta in Pulp Fiction looking for these `;`'s.

Reply:
FYI this project enforces no semicolons

Reply:
This project follows [StandardJS](https://standardjs.com/), which advises agains semicolons

Reply:
😢